### PR TITLE
Fix rerun icon on action view component

### PR DESCRIPTION
**Pull request by HesterG**
Right now rerun icon on action view component will not be seen when duration text length is long, because the wrapper `job-brief-info` has a fixed width, and the svg is squeezed. The way to fix this in this PR is to remove width and exchange position of duration text and rerun svg, and add `min-width` to job title

Before (rerun svg not shown on hover):

Screen Shot 2023-06-27 at 12 53 41

After:

Screen Shot 2023-06-27 at 12 50 59

### CODE REVIEW

**Review by wolfogre — APPROVED** [review]

**Comment by silverwind** [comment]
What happens if left text becomes too long? Will it ellipsis?

**Comment by HesterG** [comment]
> What happens if left text becomes too long? Will it ellipsis?

Yes, but duration position is squeezed.. I will need to modify this

Screen Shot 2023-06-27 at 17 25 59

**Comment by silverwind** [comment]
Check how frontpage repolist does it.

**Comment by HesterG** [comment]
> Check how frontpage repolist component does it.

Thanks, I checked it and I found by adding `min-width` it fixed the problem

Screen Shot 2023-06-27 at 17 48 37

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -610,7 +611,7 @@ export function initRepositoryActionView() {
 .job-brief-item .job-brief-info {
   display: flex;
   align-items: center;
-  width: 55px;
+  width: fit-content;
```
Comment (silverwind) [inline web_src/js/components/RepoActionView.vue]: Can it be removed? I personally never use this value, but if it's really necessary, keep it.

Comment (HesterG) [reply web_src/js/components/RepoActionView.vue]: Yes, removed it.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
I know a general solution to this icon shrinking problem, but it either requires a inline style on every SVG, or 40+ CSS rules for each pixel size, unfortunately.

**Pull request merged**